app/src/components/v-form/form-field-interface.vue

```diff
@@ -24,6 +24,8 @@
 			:collection="field.collection"
 			:field="field.field"
 			:field-data="field"
+			:values="values"
```

Comment:
`values` is already provided and can be injected in sub components of the form using `inject('values')` 
Can you rather use that.

Reply:
But we don't all use vue 😄 

Assigning values like that allows a setting on a vanilla web component to participate in the fun.